gtda/homology/grids.py

```diff
@@ -9,12 +9,12 @@
 from ._utils import _pad_diagram
 from ..utils.validation import validate_params
 
-from ..externals.python import CubicalComplex
+from ..externals.python import CubicalComplex, PeriodicCubicalComplex
 
 
 class CubicalPersistence(BaseEstimator, TransformerMixin):
-    """`Persistence diagrams <https://giotto.ai/theory>`_ resulting from
-    `filtrations of Cubical complex <https://giotto.ai/theory>`_.
+    """`Persistence diagrams <https://giotto.ai/theory>`_ resulting from a
```

Comment:
Remove "a"

Reply:
Ok, I will leave the wording to you @ulupo  - it was me who suggested adding 'a' in the first place :( Sorry, @gtauzin !